### PR TITLE
Force same timestamp for all discovers and same batch mutations

### PR DESCRIPTION
**Pull request by ssivov**
Having same timestamp on multiple events allows for easier batch processing on the backend

### CODE REVIEW

**Comment by toby-walker** [comment]
The PR looks fine but it's missing why you are making this change. Can you explain the context.?

**Comment by ssivov** [comment]
Discover and mutations are designed to be atomic operations. Giving them different timestamps creates possibilities of unexpected states in processing logic (especially visualization).

---
In reply to: [456186003](https://github.com/Microsoft/clarity-js/pull/122#issuecomment-456186003) [](ancestors = 456186003)

File: src/plugins/layout.ts [hunk 1]
```diff
@@ -241,7 +243,10 @@ export default class Layout implements IPlugin {
     let moves = summary.movedNodes.map(this.processMutation.bind(this, Action.Move));
     let updates = summary.updatedNodes.map(this.processMutation.bind(this, Action.Update));
     let removes = summary.removedNodes.map(this.processMutation.bind(this, Action.Remove));
-    let all = [].concat(inserts, moves, updates, removes);
+    let all: IEventData[] = [].concat(inserts, moves, updates, removes);
+    for (let i = 0; i < all.length; i++) {
```
Comment (toby-walker) [inline src/plugins/layout.ts]: Can you add a comment here that includes your response to my question about "why this PR?" please.

Comment (ssivov) [reply src/plugins/layout.ts]: Done

**Pull request merged**